### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by markhollis**
## Issue I worked on

see https://issues.openmrs.org/browse/CALC-54

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by dkayiwa** [comment]
Did you get a chance to look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by markhollis** [comment]
I did yes, that is where I learned how to squash the commits. Did it work out okay?

**Comment by dkayiwa** [comment]
Did you look at the JIRA workflow on the same page?

**Comment by markhollis** [comment]
I don't think I see those buttons on my page - should I be seeing a 'Request Code Review' button on my page? I am also unable to edit the status to show 'Ready for Testing'.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,71 @@
...
+## Authors
+
+See the list of [contributors](https://github.com/openmrs/openmrs-module-calculation/graphs/contributors) who participated in this project.
+
+## Project Resources
+
+[Wiki page](https://wiki.openmrs.org/display/docs/Calculation+Module)
+
+## License
+This project is licensed under the OpenMRS Public License Version 1.0 (the "License"). Obtain a copy of the License at
```
Comment (dkayiwa) [inline README.md]: This is an old version of the license.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,71 @@
...
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the Calculation Module on a live system.
+
+### Prerequisites
+
+Follow instructions on OpenMRS web site to download and install the OpenMRS SDK, found at https://wiki.openmrs.org/display/docs/OpenMRS+SDK+Step+By+Step+Tutorials
+
+### Installing
+
+To install the Calculation Module you will need to download and install the module from github. Follow the following instructions to do this.
```
Comment (dkayiwa) [inline README.md]: Compiled version of the module are got from: https://addons.openmrs.org/#/show/org.openmrs.module.calculation

**Comment by markhollis** [comment]
The commits have been squashed. Would you mind taking a look at our most previous commit?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,71 @@
...
+http://localhost:8080/openmrs/initialsetup/
+```
+
+for a first time setup.
+
+If you have any trouble getting your OpenMRS server running, please go to https://wiki.openmrs.org/display/docs/OpenMRS+SDK#OpenMRSSDK-Settingupservers
+
+## Running the tests
+
+Tests automatically run with the installation of the Calculation Module. If the module is already installed, simply move to the openmrs/openmrs-module-calculation folder and use the following instruction to reinstall the module, running the test cases.
```
Comment (dkayiwa) [inline README.md]: Does this mean that tests are run when an end user is installing the module?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,71 @@
...
+```
+/openmrs/modules/
+```
+
+Drag your .omod file into this folder, if it does not already contain one.
+
+From here, run your OpenMRS server and navigate to 
+
+```
+http://localhost:8080/openmrs/initialsetup/
```
Comment (dkayiwa) [inline README.md]: How about those whose instance name is not openmrs and are not running on port 8080?

File: README.md (hunk 3 above)
Comment (HallieH) [reply README.md]: Yes, the test cases run when the module runs the installation instructions, but not when the simply download the .omod file from https://addons.openmrs.org/#/show/org.openmrs.module.calculation. Is there a different way we should write this to make that more clear, or does that clear things up?

**Comment by HallieH** [comment]
We made changes that you suggested and that we thought were necessary. Could you take another look?

**Pull request merged**